.github/workflows/commits.yml

```diff
@@ -0,0 +1,48 @@
+name: Commits
+on:
+  - pull_request
+
+permissions:
+  contents: read
+
+jobs:
+  cla-check:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    name: Canonical CLA signed
+    steps:
+      - name: Check if CLA signed
+        uses: canonical/has-signed-canonical-cla@v2
+
+  dco-check:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read  # for tim-actions/get-pr-commits to get list of commits from the PR
+    name: Signed-off-by (DCO)
+    steps:
+    - name: Get PR Commits
+      id: 'get-pr-commits'
+      uses: tim-actions/get-pr-commits@master
```

Comment:
I know we use it like this in other repositories, but reading it back now with the tj-actions supply chain attack in recent memory, should we perhaps pin this at a specific commit hash instead of using a branch or tag?

Reply:
sounds very reasonable 👍